### DISCUSSION TITLE
LIFX lifx.set_state support

### DISCUSSION THREAD

**Discussion by Zipties** [basnijholt/adaptive-lighting#258] [Ideas]
When turning my LIFX lights on it can take AL a few seconds to apply the change. This can be jarring especially late at night.

LIFX supports changing the brightness and color of a bulb without changing its power state with the lifx.set_state service. It would be great if AL could be configured to take advantage of that.

**Comment by Djelibeybi** [comment]
See #576 for a way to use `lifx.set_state` with Adaptive Lighting.
